packages/paste-core/components/tooltip/src/index.tsx

```diff
@@ -38,38 +35,47 @@ const StyledTooltip = React.forwardRef<HTMLDivElement, BoxProps>(({style, ...pro
 export interface TooltipStateReturn extends TooltipPrimitiveStateReturn {
   [key: string]: any;
 }
-export interface TooltipProps extends TooltipPrimitiveInitialState {
+export interface TooltipProps extends Pick<BoxProps, 'element'>, TooltipPrimitiveInitialState {
```

Comment:
Does the order matter here?

`export interface TooltipProps extends Pick<BoxProps, 'element'>, TooltipPrimitiveInitialState {`
vs.
`export interface TooltipProps extends TooltipPrimitiveInitialState, Pick<BoxProps, 'element'> {`

Reply:
I'm not sure; the first time I did it I think I missed a comma or something so it gave me an error, and when it worked in this order, I kept it 😬 . Just changed it to have Pick<BoxProps, 'element'> be last for consistency though (which, it works now 😅 )